src/MainView.vala

```diff
@@ -65,7 +65,7 @@ public class Power.MainView : Gtk.Grid {
         if (battery.is_present ()) {
             var wingpanel_power_settings = new GLib.Settings ("io.elementary.desktop.wingpanel.power");
 
-            var show_percent_label = new Gtk.Label (_("Show percentage:")) {
+            var show_percent_label = new Gtk.Label (_("Show battery percentage in Panel:")) {
```

Comment:
Do you want to be more specific with the naming here? i.e. Wingpanel?

Reply:
@pongloongyeat No. We never use `WingPanel` in the OS or other user-facing copy; it's always referred to as `the Panel`.